python/tk_multi_about/context_browser.py

```diff
@@ -78,7 +84,13 @@ def process_result(self, result):
             i = self.add_item(browser_widget.ListItem)
             details = []
             details.append("<b>Project %s</b>" % d.get("name"))
-            details.append( d.get("sg_description") if d.get("sg_description") else "No Description" )            
+            details.append( d.get("sg_description") if d.get("sg_description") else "No Description" )
+
+            link_color = self._app.engine.style_constants.get("SG_HIGHLIGHT_COLOR","#18A7E3")
```

Comment:
i suggest you just do `self._app.engine.style_constants["SG_HIGHLIGHT_COLOR"]`. Why the fallback? 

Also pls bump the min core to make sure the app requires the core that added `.style_constants` (0.17.something)

Reply:
min core is set at 
`requires_core_version: "v0.18.45"`
So should be good, looks like they were added in `v0.16.15`, but I must admit I hadn't considered that, so good thing for me to think about in the future.